README.md

```diff
@@ -73,6 +73,9 @@ if (cordova.platformId == 'android') {
 }
 ```
 
+### Browser Support
+Only a limited selection of browsers offer support for setting status bar colors.
```

Comment:
Can you expand on which browsers offer this support? I feel like that's valuable information for this documentation.